app/src/hooks/useDataLoader.test.tsx

```diff
@@ -0,0 +1,23 @@
+import { render } from '@testing-library/react';
+import React from 'react';
+import useDataLoader from './useDataLoader';
+
+interface ITestComponentProps {
+  callback: (() => any) | null | undefined;
+  
+}
+
+const TestComponent: React.FC<ITestComponentProps> = (props) => {
+
+  return <></>;
+};
+
+describe('useDataLoader', () => {
```

Comment:
We can also tackle these tests in a separate branch, and then we can get this PR merged. Up to you either way, as we have lots of time left in the sprint still.